Fixed multiple levels replacing superfluous slashes

steve962 opened the pull request:
Closes #55 
Closes https://github.com/goetas-webservices/xsd-reader/issues/57

goetas commented:
@steve962  I'm sorry if it took me that long to reply here, i'm ok with the changed, can you just fix the failing tests please? (and then I plan to merge it)

steve962 commented:
@goetas I haven't done pull requests on here, and unfortunately thought it was tied to the commit, not the branch... so the unrelated change for the progress report callback was accidentally included in this PR and broke things.   I moved it to it's own branch and reverted the change in the master branch - so it passes now.

goetas commented:
Sounds good, thanks for your work
